apps/test/util/testUtils.js

```diff
@@ -245,3 +246,20 @@ function getBooleanPermutation(n, numberOfBooleans) {
 function zeroPadLeft(string, desiredWidth) {
   return ('0'.repeat(desiredWidth) + string).slice(-desiredWidth);
 }
+
+/**
+ * Call in any mocha scope to make console.error() throw instead
+ * of log.  Useful for making tests fail on a React propTypes validation
+ * failures.
```

Comment:
I understand why this works the way it does, but I still wouldn't necessarily expect to call a function within my test case and have it change how my tests work. Given that the proper usage of this is just a bit odd, it seems appropriate to add either an example or an `@see testUtilsTest.throwOnConsoleErrors` in this comment.